docs/src/pages/[platform]/theming/theme-provider/react.mdx

```diff
@@ -0,0 +1,89 @@
+import { Flex } from '@aws-amplify/ui-react';
+import { Example, ExampleCode } from '@/components/Example';
+
+import { BasicExample, CustomThemeExample } from './examples';
+
+## Usage
+
+Import the `ThemeProvider` and wrap your application with it:
+
+```jsx
+import { ThemeProvider } from '@aws-amplify/ui-react';
+
+export const App = () => (
+  <ThemeProvider>
+    <YourApplication />
+  </ThemeProvider>
+);
+```
+
+After wrapping your application in the `ThemeProvider`, you have access to all theme values in your components. To style the components in your app, you can either:
+1. Get the theme `tokens` through the `useTheme` hook (e.g., `tokens.colors.blue[80]`)
+2. Reference the theme `tokens` directly in style props (e.g., `"purple.80"`)
+
+<Example>
+  <Flex>
+    <BasicExample />
+  </Flex>
+
+  <ExampleCode>
+    ```tsx file=./examples/BasicExample.tsx
+    ```
+    
+  </ExampleCode>
+</Example>
+
+### theme
+
+To create and use your own custom theme, you may pass a [theme object](https://ui.docs.amplify.aws/react/theming#theme-object) to the `theme` prop on the `ThemeProvider`. 
+
+- [Theming overview](./theming)
+
+<Example>
+  <CustomThemeExample />
+
+  <ExampleCode>
+    ```tsx file=./examples/CustomThemeExample.tsx
+    ```
+    
+  </ExampleCode>
+</Example>
+
+### colorMode
+
+The `ThemeProvider` accepts a `colorMode` prop which can be `light`, `dark`, or `system`.
+
+See the [Dark mode documentation](./dark-mode) for a detailed explanation of how to use the `colorMode` prop. 
```

Comment:
Did you test the link `./dark-mode`? It might prevent future errors if we use `theming/dark-mode`

Reply:
Yes I tested the link `./dark-mode`